x/payment/client/testutil/integration_test.go

```diff
@@ -83,19 +83,7 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForData() {
 				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
 				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(2))).String()),
 				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
-			},
-			false, 0, &sdk.TxResponse{},
-		},
-		{
-			"valid transaction list of square sizes",
-			[]string{
-				hexNS,
-				hexMsg,
-				fmt.Sprintf("--from=%s", username),
-				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
-				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(2))).String()),
-				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
-				fmt.Sprintf("--%s=%s", paycli.FlagSquareSizes, "256,128,64"),
+				fmt.Sprintf("--%s=%s", paycli.FlagSquareSizes, "2,4,8,16"),
```

Comment:
Q: shouldn't we at least test for 32 and 64 squares here as well?

Reply:
Now that we are efficiently filling the square and actually using a square size other than 64, messages have to be signed over accordingly. Since there is only a single transaction in this block, the square will be small.

You're right though, we should test with a larger message here. I tried and ran into #379 when adding the flag to increase the gas limit.

#400